From: Sebastian Ott <sebott@linux.vnet.ibm.com>
Subject: revert patches.arch/ccwgroup-fix-uevent-vs-dev-attrs-race.patch
Patch-mainline: no
Git-commit: -
References: bnc#790870

This reverts patches.arch/ccwgroup-fix-uevent-vs-dev-attrs-race.patch

Signed-off-by: Sebastian Ott <sebott@linux.vnet.ibm.com>
Acked-by: John Jolly <jjolly@suse.de>
---
 drivers/s390/cio/ccwgroup.c |    3 ---
 1 file changed, 3 deletions(-)

--- a/drivers/s390/cio/ccwgroup.c
+++ b/drivers/s390/cio/ccwgroup.c
@@ -314,15 +314,12 @@ int ccwgroup_create_from_string(struct d
 
 	dev_set_name(&gdev->dev, "%s", dev_name(&gdev->cdev[0]->dev));
 	gdev->dev.groups = ccwgroup_attr_groups;
-	dev_set_uevent_suppress(&gdev->dev, 1);
 	rc = device_add(&gdev->dev);
 	if (rc)
 		goto error;
 	get_device(&gdev->dev);
 	rc = __ccwgroup_create_symlinks(gdev);
 	if (!rc) {
-		dev_set_uevent_suppress(&gdev->dev, 0);
-		kobject_uevent(&gdev->dev.kobj, KOBJ_ADD);
 		mutex_unlock(&gdev->reg_mutex);
 		put_device(&gdev->dev);
 		return 0;
